### PR TITLE
Getfilemeta

### PR DESCRIPTION
**Pull request by Kaspermn33**

### CODE REVIEW

File: src/getfilemeta.ml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            version = file.version; 
+            versionChanged = 1; 
+            timestamp = (string_of_int file.msTimestamp)    
+        }
+
+
+(*let matchBodyWithExpectedResult (bodyResult: resultData) (userId: int) (fileId: int) (state: Orbit.system) : bool =
+  let expectedData = getExpectedResultData userId fileId state in
+
+  if(compare expectedData bodyResult) != 0 then false else true*)
```
Comment (dueruen) [inline src/getfilemeta.ml]: Use or not?

File: src/getfilemeta.ml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        Http_common.create_response ~content_type: (Some "application/json") Http_common.HttpOk
+;;
+
+let matchResults (userId: int) (fileId: int) (state: Orbit.system) (body: string) (requestResult: Http_common.response) : bool = 
+    let expectedResultHeaders = getExpectedResultHeaders userId fileId state in
+    if (compare expectedResultHeaders requestResult) != 0 then false else 
+
+    (*if (requestResult.status_code != Http_common.HttpOk) then false else
+    let expectedResultData = getExpectedResultData userId fileId state in
+    if (compare expectedResultData body) != 0 then false else true*)
```
Comment (dueruen) [inline src/getfilemeta.ml]: Use or not?

File: src/machine.ml [hunk 3]
```diff
@@ -21,7 +22,9 @@ struct
     | Move_File of ((int * int * int) * (int * string * int32))
     | Create_File of int * int * string * int32 
     | Delete_File of int * int * int
-    | Delete_Dir of int * int * int [@@deriving show { with_path = false }] 
+    | Delete_Dir of int * int * int 
+    | Get_File_Meta of int * int [@@deriving show { with_path = false }]
+    (* | Create_File of (int) (int) (int) (int) *)
```
Comment (grammeaway) [inline src/machine.ml]: Yeet or keep?

**Review by grammeaway — CHANGES_REQUESTED** [review]

**Review by grammeaway — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**